core/metacling/src/TCling.cxx

```diff
@@ -5877,7 +5877,8 @@ static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_nam
 
    static bool sFirstRun = true;
    // We don't want to do directory_iterator several times because it contains HDD access
-   static std::set<std::string> sLibraries;
+   static std::vector< std::pair<uint32_t, std::string> > sLibraries;
+   static std::vector<std::string> sPaths;
```

Comment:
We can collapse s`Path` and `alreadyLookedPath` into llvm::SetVector data structure.

Reply:
sPath is static variable which we want to store but alreadyLookedPath is a temporary variable